src/components/Lakritzel/Footer.js

```diff
@@ -1,62 +1,58 @@
 import styled from 'styled-components'
-import { Link } from 'react-router-dom'
-import cocktailImg from '../Lakritzel/Assets/Logo.jpg'
-import bottleImg from '../Lakritzel/Assets/painted-bottle.jpg'
-import shot from '../Lakritzel/Assets/Pinnchen.jpg'
-import portfolio from '../Lakritzel/Assets/Portfolio.jpg'
+import { NavLink } from 'react-router-dom'
+import cocktailImg from '../Lakritzel/Assets/Logo.svg'
+import bottleImg from '../Lakritzel/Assets/painted-bottle.svg'
+import shot from '../Lakritzel/Assets/Pinnchen.png'
+import portfolio from '../Lakritzel/Assets/Portfolio.png'
 
 function Footer() {
   return (
     <Wrapper>
-      <Link to="/">
-        <Product src={bottleImg} alt="Products" />
-      </Link>
-      <Link to="/campaign">
-        <Motiv src={cocktailImg} alt="Motive" />
-      </Link>
-      <Link to="/cocktails">
-        <Cocktails src={shot} alt="Cocktails and Shots" />
-      </Link>
-      <Link to="/kritzelkopf">
-        <DoodleHead src={portfolio} alt="The Man behind the Monster" />
-      </Link>
+      <Navigation exact to="/" activeClassName="active">
+        <Image src={bottleImg} alt="Products" />
+      </Navigation>
+      <Navigation exact to="/campaign" activeClassName="active">
+        <Image src={cocktailImg} alt="Motive" />
+      </Navigation>
+      <Navigation exact to="/cocktails" activeClassName="active">
+        <Image src={shot} alt="Cocktails and Shots" />
+      </Navigation>
+      <Navigation exact to="/kritzelkopf" acrtiveClassName="active">
+        <Image src={portfolio} alt="The Man behind the Monster" />
+      </Navigation>
     </Wrapper>
   )
```

Comment:
I think it would be better to rename `Wrapper` in `Navigation` and `Navigation` in `NavLink` for example. `Navigation` is also an html tag (`nav`) that stores the navigation links.